static/app/services.js

```diff
@@ -0,0 +1,6 @@
+(function() {
+    'use strict';
```

Comment:
To be honest, I had a service here and removed it. But we can keep something like this around so we have somewhere to put services...